cabot/cabotapp/alert.py

```diff
@@ -33,13 +35,18 @@ def __unicode__(self):
         return u'%s' % (self.title)
 
 
-def send_alert(service, duty_officers=None):
+def send_alert(service, duty_officers=None, fallback_officers=None):
     users = service.users_to_notify.filter(is_active=True)
     for alert in service.alerts.all():
         try:
             alert.send_alert(service, users, duty_officers)
-        except Exception:
-            logging.exception('Could not sent ' + alert.name + ' alert')
+        except Exception as e:
+            logging.exception('Could not sent {} alert: {}'.format(alert.name, e))
```

Comment:
This may be sufficient
`logger.exception("Could not sent alert: {}'.format(alert.name))`

The exception will get logged automatically.

Minor. your call